client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,16 +44,12 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Properties panel quality of life improvements</h4>
-          We removed automatic (magic) sorting from all properties panels. Additional minor improvements ensure that implementing Camunda 7 and 8 is simpler than ever.
+          <h4>Enhanced Form Component Documentation</h4>
+          We've added convenient documentation links directly within the properties panel for Form components. This allows you to quickly find information about the functions and configurations when you need it.
         </li>
         <li>
-          <h4>More robust BPMN diagram import</h4>
-          Various improvements ensure that the modeler can import even what is (accidentally) broken.
-        </li>
-        <li>
-          <h4>New modeling experience inside</h4>
-          You can now enable our reworked context pad via <a href="https://docs.camunda.io/docs/next/components/modeler/desktop-modeler/flags/#enable-new-context-pad" target="_blank" rel="noopener">a feature toggle</a>.
+          <h4>Improved Accessibility</h4>
```

Comment:
```suggestion
          <h4>Improved accessibility</h4>
```

Reply:
Sentence case

Reply:
addressed in [049a158](https://github.com/camunda/camunda-modeler/pull/4384/commits/049a1588313c7b31edbecb51d039ecb4e7b4132a)